README.md

```diff
@@ -52,7 +52,7 @@ Generate protobuf code and out.pb (used for cloud endpoints deployment).
 
     go run server/main.go \
       --bq_dataset=google.com:datcom-store-dev.dc_v3_clustered \
-      --bt_table=dc24 \
+      --bt_table=dc25 \
```

Comment:
Please confirm with tjann on this?

Reply:
Reverting this change. Defer to tjann on changing these.